src/program/lwaftr/tests/subcommands/run_nohw_test.py

```diff
@@ -0,0 +1,77 @@
+"""
+Test the "snabb lwaftr run_nohw" subcommand.
+"""
+
+import unittest
+
+from random import randint
+from subprocess import call, check_call
+from test_env import DATA_DIR, SNABB_CMD, BaseTestCase
+
+class TestRun(BaseTestCase):
+
+    program = [
+        str(SNABB_CMD), 'lwaftr', 'run_nohw',
+    ]
+    cmd_args = {
+        '--duration': '1',
+        '--bench-file': '/dev/null',
+        '--conf': str(DATA_DIR / 'icmp_on_fail.conf'),
+        '--inet-if': '',
+        '--b4-if': '',
+    }
+    veths = []
+
+    @classmethod
+    def setUpClass(cls):
+        cls.create_veth_pair()
+
+    @classmethod
+    def create_veth_pair(cls):
```

Comment:
Since `create_veth_pair` is only called by `setUpClass`, it probably doesn't need to be a separate classmethod. What it does is explained by comments anyway.